guides/source/upgrading_ruby_on_rails.md

```diff
@@ -272,6 +272,78 @@ You can invalidate the cache either by touching the product, or changing the cac
 <% end %>
 ```
 
+#### Migrating from Marginalia
+
+Active Record ships with the same default configuration as Marginalia. For basic use, enable the feature in your application configuration:
+
+```ruby
+# config/application.rb
```

Comment:
We typically don't want to overload the application configuration, so can we put the configurations in `config/initializers/query_log_tags.rb` instead? Later we can have new Rails 7 apps generate that by default.

Reply:
Sorry old habits - in general do we (app developers) not change the application config any more? 

Slightly thrown by this. If I wanted to enable the feature without any further customization of the tags, changing the config option in `config/application.rb` seems more appropriate than adding a new initializer, but I accept that I may be behind the times here! :)

Reply:
Sorry to bring this up again @kaspth but I'm still thrown by where developers are expected to enable a feature. 

If someone wishes to enable default behaviour, would we still guide them towards creating an initializer?

If so, would they go for:

```
Rails.application.config.active_record.query_log_tags_enabled = true
```